tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -54,6 +76,19 @@ function delete_runtimeclasses() {
 	echo "Deleting the runtime classes"
 
 	for shim in "${shims[@]}"; do
+		if [[ "${USING_NFD}" == "true" ]]; then
+
+			# Safety check to ensure NFD worker is deployed and is running
+			[[ $(kubectl get pods -n node-feature-discovery | grep nfd-worker) ]] || die "NFD worker was not found in the cluster"
```

Comment:
Suppose we installed the runtimeclasses correctly. On deleting, NFD is not working...is there any collateral effect of bailing out here?